docs/en/UI/Angular/Checkbox-Component.md

```diff
@@ -0,0 +1,39 @@
+# Checkbox Component
+
+The ABP Checkbox Component is a reusable form input component for checkbox type.
+
+It takes `label`, `labelClass` (default form-check-label), `checkboxId`,`checkboxReadonly`,`checkboxReadonly` (default form-check-input) and `checkboxStyle` as input to customize component. Also, it emits `checkboxBlur` and `checkboxFocus` events when component gain or lose focus.
```

Comment:
We can use bulleted list for inputs & outputs. Add to descriptions near to list item

# Example

## Inputs
- `label`
- `labelClass (default: form-check-label)`
- etc..

## Outputs
- `checkboxBlur`
- etc..